historical/views/containers/chatroom/MessageInteractive.vue

```diff
@@ -0,0 +1,205 @@
+    // [STATUS_FAILED]: L('Proposal from {from} to change the {setting} is rejected.', options),
+    // [STATUS_EXPIRED]: L('Proposal from {from} to change the {setting} is expired.', options),
+    [STATUS_EXPIRING]: L('Proposal from {from} to change the {setting} is expiring.', options)
+  })
+
+  const interactiveMessages = {
+    [PROPOSAL_INVITE_MEMBER]: {
+      // [STATUS_OPEN]: L('{from} wants to add {member} to the group.', options),
+      // [STATUS_PASSED]: L('Proposal from {from} to add {member} is accepted.', options),
+      // [STATUS_FAILED]: L('Proposal from {from} to add {member} is rejected.', options),
+      // [STATUS_EXPIRED]: L('Proposal from {from} to add {member} is expired.', options),
+      [STATUS_EXPIRING]: L('Proposal from {from} to add {member} is expiring.', options)
+    },
+    [PROPOSAL_REMOVE_MEMBER]: {
+      // [STATUS_OPEN]: L('{from} wants to remove {member} from the group.', options),
+      // [STATUS_PASSED]: L('Proposal from {from} to remove {member} is accepted.', options),
+      // [STATUS_FAILED]: L('Proposal from {from} to add {member} is rejected.', options),
+      // [STATUS_EXPIRED]: L('Proposal from {from} to add {member} is expired.', options),
+      [STATUS_EXPIRING]: L('Proposal from {from} to remove {member} is expiring.', options)
+    },
+    [PROPOSAL_GROUP_SETTING_CHANGE]: {
+      mincomeAmount: settingChangeMessages(options),
+      distributionDate: settingChangeMessages(options)
+    },
+    [PROPOSAL_PROPOSAL_SETTING_CHANGE]: {
+      votingRule: settingChangeMessages(options),
+      votingSystem: settingChangeMessages(options)
+    },
+    [PROPOSAL_GENERIC]: {
+      // [STATUS_OPEN]: L('{from} created a proposal. "{title}"', options),
+      // [STATUS_PASSED]: L('Proposal from {from} is accepted. "{title}"', options),
+      // [STATUS_FAILED]: L('Proposal from {from} is rejected. "{title}"', options),
+      // [STATUS_EXPIRED]: L('Proposal from {from} is expired. "{title}"', options),
+      [STATUS_EXPIRING]: L('Proposal from {from} is expiring. "{title}"', options)
+    }
+  }
+
+  return get(interactiveMessages, [proposalType, options.settingType, variant].filter(key => !!key))
+}
+
+const proposalStatus = (proposal) => {
+  const options = {}
+  if (proposal.variant === STATUS_EXPIRING) {
+    options['date'] = humanDate(proposal.expires_date_ms, { month: 'short', day: 'numeric', year: 'numeric' })
+  }
+  return {
+    // [STATUS_OPEN]: L('New proposal'),
+    // [STATUS_PASSED]: L('Proposal Accepted'),
+    // [STATUS_FAILED]: L('Proposal rejected'),
+    // [STATUS_EXPIRED]: L('Proposal expired'),
+    [STATUS_EXPIRING]: L('Proposal expiring on {date}', options)
+  }[proposal.variant]
+}
+
+const proposalSeverity = {
+  // [STATUS_OPEN]: 'is-info',
+  // [STATUS_PASSED]: 'is-success',
+  // [STATUS_FAILED]: 'is-danger',
+  // [STATUS_EXPIRED]: 'is-neutral',
+  [STATUS_EXPIRING]: 'is-warning'
+}
```

Comment:
What are all of these commented out lines doing here? 🤔

Reply:
While I was working in this PR, I noticed that only `STATUS_EXPIRING` was reachable and others were not. Because we were creating only one Interactive message when the proposal was expiring. That was why I commented them out.

But I think, since we are moving this file to the `/history` folder, those should not be commented out.